web/templates/buddy_list_tooltip_content.hbs

```diff
@@ -6,9 +6,9 @@
         {{/if}}
     </span>
     {{#if second_line}}
-    <br /><span class="tooltip-inner-content">{{second_line}}</span>
+        <p class="tooltip-inner-content">{{second_line}}</p>
```

Comment:
I wonder if these shouldn't be `<div>` tags rather than `<p>` takes. Maybe it's an edge case not worth worrying about, but the range of HTML elements that should appear within a `<p>` is a lot smaller than what's permissible in a `<div>`.

Additionally, screen readers will often introduce a long, annoying pause after reading the content of a paragraph element.

Reply:
That said, perhaps the first lines should be `<div>` elements, too, especially because we're really calling for a block presentation here, and `<span>` elements default to inline. Not that you couldn't muck about with `display` in CSS, but it seems to me better practice to express our intention as to that sort of thing in HTML wherever possible.

Reply:
@karlstolley, I definitely agree with you. Using `<div>` tags in place of the `<p>` tags does make a lot of sense since the outcome that we want to achieve is just block elements so that we can avoid the use of the breaks. The additional semantic meaning of the `<p>` tags doesn't really apply to our situation here, and on top of that, since the `<p>` tag cannot contain other block-level elements, it does limit what we are able to put inside the tooltip contents. (we don't really use such a tooltip at the present, but it doesn't hurt in case we do need it at some point.)